src/wallet/feebumper.cpp

```diff
@@ -223,8 +230,9 @@ Result CreateRateBumpTransaction(CWallet* wallet, const uint256& txid, const CCo
 
     // Fill in recipients(and preserve a single change key if there is one)
     std::vector<CRecipient> recipients;
+    const auto* change_output = reduce_output > -1 && (size_t)reduce_output < wtx.tx->vout.size() ? &wtx.tx->vout[reduce_output] : nullptr;
```

Comment:
`CreateTotalBumpTransaction` returns an error if `reduce_output` is out of range. This code just ignores it. Probably should be consistent.

Reply:
"change" is really over-loaded here, please call it something else

Reply:
@luke-jr Agreed - now consistent.

@instagibbs Renamed to `reduce_output_vout`.